src/ui/WorkInProgressRoot.tsx

```diff
@@ -286,7 +286,7 @@ export function WorkInProgressRoot(): React.ReactElement {
   if (isCreateProgramWork(Player.currentWork)) {
     const create = Player.currentWork;
     const completion = (create.unitCompleted / create.unitNeeded()) * 100;
-
+    const rate = ((create.unitNeeded() - create.unitCompleted) / create.unitRate) * CONSTANTS.MilliPerCycle;
```

Comment:
These should be called "remaining", since they're times not rates.

Reply:
changed to `remainingTime`